docs/application/native/guides/device/resource-monitor.md

```diff
@@ -0,0 +1,302 @@
+# Resource Monitor
+
+You can monitor the various resources in a unified APIs with [resource type](#resource_types), [attributes](#resource_attributes), and some [controls](#resource_controls).
+
+The basic procedures for getting monitoring data are simple and unified as below.
+
+At first, setup environment what resources and attributes are monitored:
+
+1. Create resource instance with the corresponding [resource type](#resource_types).
+
+2. If required, set [resource control](#resource_controls) to specify the target among multi-instantiable resources.
+
+3. Set interesting [attributes](#resource_attributes) for created resource instance.
+
+Now, it is ready to get monitoring data. Repeat the next two step when application want to get resource attributes:
+
+4. Request update the individual resource or all resources at once.
+
+5. Get the attribute value along with the type of attributes using APIs below:
+
+**Table: Resource Monitor APIs for getting attribute values**
+
+ | Functions                                                                     | Description                                                |
+ |-------------------------------------------------------------------------------|------------------------------------------------------------|
+ | `resource_monitor_get_value_[int\|int64\|uint32\|uint64\|double\|string]`     | Get the attribute data                                     |
+ | `resource_monitor_get_array_[int\|int64\|uint32\|uint64\|double\|string]`     | Get array-type attribute data with the corresponding type  |
+
+
+## Prerequisites
+
+To enable your application to use the Resource Monitor functionalities:
+
+1. To use the [Resource Monitor](../../api/wearable/latest/group__CAPI__SYSTEM__RESOURCE__MONITOR__MODULE.html) API, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
```

Comment:
This link appears to be broken but I think that is because the XML file for Tizen 7.0 hasn't been updated yet.

Reply:
@mijin85cho, hello.
Will this broken link be an issue?

Reply:
Hello, @omar-srbd 
As you said, there is no Tizen 7.0 Native API yet and it's the reason of the broken link.
For now we can proceed considering the API file will be added later.
When Tizen 7.0 is published, please check this link at that time again.

Reply:
@mijin85cho, thanks for confirming the matter. (: